### PR TITLE
Expand documentation for eval mode

### LINKED ISSUE #253: How to enter debug mode again after changing a variable in REPL?
Hi!

I am really stuck and just hope, someone will answer my dumb question.
I am trying to debug in julia for the first time... I'm an engineer and not used to using the console that much.

So I watched this debugger demo:
https://www.youtube.com/watch?v=SU0SmQnnGys

It took me a lot of further googling to realize that he uses a ` to get from the debug mode to normal mode, in order to change a variable.
Now that he's changed the variable, he uses a magic trick to enter debug mode again...

I would like to know, how he does it?
Thanks a lot already to anyone reading this and hopefully answering?

### PR DESCRIPTION
**Pull request by timholy**
Includes how one exits eval mode. Fixes #253.

### CODE REVIEW

**Comment by KristofferC** [comment]
Should probably also be added to the README (a bit of unfortunate documentation duplication right now).

File: src/commands.jl [hunk 1]
```diff
@@ -294,7 +293,10 @@ function execute_command(state::DebuggerState, ::Union{Val{:help}, Val{:?}}, cmd
                 - `bp on/off throw` - turn on or off break on throw\\
 
 
-            An empty command will execute the previous command.""")
+            An empty command will execute the previous command.
+        
+            Hit ``` to enter "evaluation mode," where any expression you type is executed in the debug context.
```
Comment (pfitzseb) [inline src/commands.jl:298]:
```suggestion
            Hit `` ` `` to enter "evaluation mode," where any expression you type is executed in the debug context.
```

File: src/commands.jl [hunk 2]
```diff
@@ -294,7 +293,10 @@ function execute_command(state::DebuggerState, ::Union{Val{:help}, Val{:?}}, cmd
...
+            Hit backspace as the first character of the line to return to "debug mode." """)
```
Comment (pfitzseb) [inline src/commands.jl:299]:
```suggestion
            Hit backspace as the first character of the line (or `^C` anywhere) to return to "debug mode." """)
```

**Comment by friederikemeier** [comment]
Nice and quick fix!

**Pull request merged**